### DISCUSSION TITLE
Welcome to generator-teams (#yoteams) Discussions!

### DISCUSSION THREAD

**Discussion by wictorwilen** [pnp/generator-teams#169] [General]
## 🎉 Hi and welcome!
  We’re using Discussions as a place to connect with other members of our this great #yoteams community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas - on what can be improved, enhanced or changed
  * Engage with and help other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by Ochuwa-sophie** [comment]
Hello! I'll like to know if yeoman for teams can scaffold using .net.

**Reply by wictorwilen** [reply]
@Ochuwa-sophie - the Yo Teams generator is as of today only scaffolding server side code for node.js and with TypeScript. We're open to suggestion and ideas and of course contributions!

**Reply by Ochuwa-sophie** [reply]
Thanks, @wictorwilen .

**Comment by 12mouna** [comment]
Hello I'm trying to build a connector using yo teams but the problem it doesn't generate for me  the configuration dialog
( i don't have this file /src/app/scripts/myConnector/MyConnectorConfig.tsx) and the Connector card ( i don't have this file too ./src/app/myConnector/myConnector.ts)

**Reply by wictorwilen** [reply]
@12mouna - what version of the generator did you use and are you following any specific instructions?
